modules/fluid_properties/doc/content/source/userobjects/LeadBismuthFluidProperties.md

```diff
@@ -0,0 +1,44 @@
+# LeadBismuthFluidProperties
+
+!syntax description /FluidProperties/LeadBismuthFluidProperties
+
+These properties are based on experiments reported in the Handbook on Lead-bismuth Eutectic Alloy and Lead
+Properties, Materials Compatibility, Thermal-hydraulics and Technologies [!citep](Fazio). 
+Most properties only depend on temperature, the fluid is considered incompressible.
+
+!table caption=Table of properties and references to the equations in [!citep](Fazio).
+| Properties                     | Equations| Equation # |
+| :----------------------------- | :------------- | :------------- |
+| Melting point $(T_{mo}, K)$    | 398 |  2.1  |
+| Density $(\rho, kg/m^3)$       | $11065 - 1.2793T$ |  2.30  |
+| Viscosity $(\mu, Pa \times s)$ | $4.94 \times 10^{-4} \times e^{\frac{754.1}{temperature}}$ |  2.83  |
+| Specific enthalpy (h, J)       | $164.8(T - T_{mo}) - 1.97 \times 10^{-2}(T^2 - T_{mo}^2) + 4.167 \times 10^6(T^3 - T_{mo}^3) + 4.56 \times 10^5(\frac{1}{T} -\frac{1}{T_{mo}})$ |  2.55  |
+| Thermal Conductivity $(k, \frac {W}{m * K})$ | $3.284 + 1.617 \times 10^{-2}T - 2.305 \times 10^{-6}T^2$ |  2.91  |
+| Isobaric Specific Heat $(cp, \frac{J}{kg * K})$ | $164.8 - 3.94 * 10^{-2}T + 1.25 \times 10^{-5}T^2 - \frac{4.56 * 10^5}{T^2}$ |  Table 2.19.3  |
```

Comment:
- $c_p$
- J/(kg-K)
- Don't use "*". Use either "\cdot" or "\times" and be consistent everywhere. Also be consistent with whether you put a multiplication symbol between the coefficient and the variable that follows, e.g, "10^{-5} T^2". I recommend not putting one there. You can add a space: "\,".